build/build.ps1

```diff
@@ -376,8 +360,22 @@
   $ubuild.DefineMethod("VerifyNuGet",
   {
     $this.VerifyNuGetConsistency(
-      ("UmbracoCms"),
-      ("Umbraco.Core", "Umbraco.Infrastructure", "Umbraco.Web.UI", "Umbraco.Examine.Lucene", "Umbraco.PublishedCache.NuCache", "Umbraco.Web.Common", "Umbraco.Web.Website", "Umbraco.Web.BackOffice", "Umbraco.Cms.Persistence.Sqlite", "Umbraco.Cms.Persistence.SqlServer"))
+      @(),
+      (
+          "Umbraco.Cms",
+          "Umbraco.Cms.Persistence.Sqlite",
+          "Umbraco.Cms.Persistence.SqlServer",
+          "Umbraco.Cms.StaticAssets",
+          "Umbraco.Core",
+          "Umbraco.Examine.Lucene",
+          "Umbraco.Infrastructure",
+          "Umbraco.PublishedCache.NuCache",
+          "Umbraco.Templates",
```

Comment:
Apparently the VerifyNuGetConsistency method in Umbraco.Build requires all generated packages to be either a NuSpec in `build\NuSpecs` (the first parameter, now empty) or a project in `src`, which for `Umbraco.Templates` isn't the case. I'll remove the check for now, instead of moving templates into src:

```suggestion
```